Sources/Kontur.Rabbitmq/AmqpSender.cs

```diff
@@ -54,12 +54,15 @@ public AmqpMessageResult Transform(IMessage message)
             catch (Exception ex)
             {
                 this.logService.Warn(ex, "Building message was failed.");
+                message.TaskCompletionSource.SetResult(false);

```

Comment:
I've set result here instead of line 71, because there is no information about the task when you create `AmqpMessageResult` with exception, but I think it's better to do on the line 71, so probably we need to rewrite his code, what do you think?